vllm/config.py

```diff
@@ -76,7 +77,18 @@ def __init__(
         self.tokenizer_revision = tokenizer_revision
         self.quantization = quantization
 
-        self.hf_config = get_config(model, trust_remote_code, revision)
+        if os.environ.get("VLLM_USE_MODELSCOPE", False):
+            # download model from modelscope
```

Comment:
# download model from ModelScope hub, use on-demand import for compatability.

Reply:
done